src/windows/BarcodeScannerProxy.js

```diff
@@ -243,7 +246,19 @@ module.exports = {
             // rotate the preview video
             var videoEncodingProperties = capture.videoDeviceController.getMediaStreamProperties(Windows.Media.Capture.MediaStreamType.videoPreview);
             videoEncodingProperties.properties.insert(ROTATION_KEY, rotDegree);
-            return capture.videoDeviceController.setMediaStreamPropertiesAsync(Windows.Media.Capture.MediaStreamType.videoPreview, videoEncodingProperties);
+
+            try {
+                return capture.videoDeviceController.setMediaStreamPropertiesAsync(Windows.Media.Capture.MediaStreamType.videoPreview, videoEncodingProperties);
+            } catch (e) {
+                if (e.number !== PHOTO_SEQUENCE_IS_IN_PROGRESS) {
```

Comment:
This error is not documented anywhere, I can just assume that we can't apply MediaStreamProperties while a photo sequence is in progress - but the error constant name says this for itself.